bundles/org.openhab.binding.surepetcare/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<thing:thing-descriptions bindingId="surepetcare"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+
+		<label>Sure Petcare API Bridge</label>
+		<description>The Sure Petcare Cloud API Bridge</description>
+
+		<channels>
+			<channel id="refresh" typeId="refreshType"/>
+		</channels>
+
+		<config-description>
+
+			<parameter name="username" type="text" required="true">
+				<label>Username (Email)</label>
+				<description>The username to access the Sure Petcare API</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>The password to access the Sure Petcare API</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="refreshIntervalTopology" type="integer" required="false" min="300" unit="s">
+				<label>Refresh Interval Topology</label>
+				<description>Query interval (in secs) for device topology</description>
+				<default>36000</default>
+			</parameter>
+			<parameter name="refreshIntervalStatus" type="integer" required="false" min="60" unit="s">
+				<label>Refresh Interval Pet Status</label>
+				<description>Query interval (in secs) for pet status</description>
+				<default>300</default>
+			</parameter>
+
+		</config-description>
+
+	</bridge-type>
+
+	<channel-type id="refreshType">
+		<item-type>Switch</item-type>
+		<label>Refresh</label>
+		<description>Triggers a cache refresh of everything (devices, pets etc.).</description>
+		<state readOnly="false"/>
+	</channel-type>
```

Comment:
I think it would be better for this to be a Thing action instead.

Reply:
Would a thing action not need a rule to trigger it? With the current channel, I can just wire a simple switch item to this channel and press the button to manually refresh.